### DISCUSSION TITLE
How to uninstall

### DISCUSSION THREAD

**Discussion by Kutekitty43** [xoxor4d/p2-rtx#107] [Q&A] [answered]
I decided to install this just to check it out, but I decided I would just replay the game normally, but I can't figure out how to uninstall it. I've tried following the steps in another discussion and attempting to uninstall like other mods, but both of these seem to have broken my P2 install. If anyone could help me, that would be great.
Update: I reinstalled the mod and the game seems to be starting again, although it is still the RTX version. I have also previously tried validating the game's files through Steam, which is what seemed to break the install in the first place

**Comment by Porppol** [comment, marked as answer]
uninstall portal 2
delete the other files thats in the portal 2 root
install portal 2 again
